configs/components/puppet-ca-bundle.rb

```diff
@@ -5,7 +5,7 @@
 
   java_available = true
   case platform.name
-  when /(el-(6|7)|fedora-f24)/
```

Comment:
same question for fc25/java

Reply:
I have no idea what we need the JDK for in the ca-bundle :man_shrugging:

Reply:
Oh, looks like it's just to make the CA bundle available to puppetserver... because of course we use the bundle in puppet-agent for that :weary: